pkg/apis/garden/v1beta1/defaults_test.go

```diff
@@ -153,6 +153,38 @@ var _ = Describe("#SetDefaults_Shoot", func() {
 							Expect(shoot.Spec.Kubernetes.KubeControllerManager.NodeCIDRMaskSize).To(Equal(&expectedSize))
 						})
 					})
+
+					Context("global machine image", func() {
+						globalMachineImageName := "jeOs"
+						globalMachineImageVersion := "1.1"
```

Comment:
I am just curious how these tests succeed, because the versions must be semver?

Reply:
Right, a shoot creation would fail with these version numbers.
These are just tests for the defaulting, the semver checks happen in the shoot strategy / storage validation :)

Reply:
thanks